types/assert-equal-jsx/assert-equal-jsx-tests.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import {assertEqualJSX} from 'assert-equal-jsx';
```

Comment:
The problem is that `import {assertEqualJSX} from "assert-equal-jsx"` is transpiled to `const assertEqualJSX = require("assert-equal-jsx").assertEqualJSX`, but `import assertEqualJSX = require("assert-equal-jsx")` is transpiled to just `const assertEqualJSX = require("assert-equal-jsx")`. So they do different things at runtime.